# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../kdebindings/qyoto-hotfix.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- kdebindings-4.1.3/csharp/qyoto/src/SmokeInvocation.cs.orig	2008-09-25 14:03:12.000000000 +0200
+++ kdebindings-4.1.3/csharp/qyoto/src/SmokeInvocation.cs	2008-11-30 20:57:35.000000000 +0100
@@ -17,7 +17,7 @@
 
 namespace Qyoto {
 
-	using Qyoto;
+	//using Qyoto;
 	
 	using System;
 	using System.Collections;
